packages/x-tailwindcss/src/x-tailwind-plugin/components/suggestion-group/button/layout.ts

```diff
@@ -0,0 +1,19 @@
+import { TailwindHelpers } from '../../../../types';
+
+/**
+ * Returns the `layout` variant for component `suggestion group button`.
+ *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the variant.
+ */
+// eslint-disable-next-line  @typescript-eslint/explicit-function-return-type
+export function suggestionGroupButtonLayout(helpers: TailwindHelpers) {
+  const { theme } = helpers;
+  return {
+    layout: {
```

Comment:
Can anyone find a better name? `suggestion-group-button-no-ratio`?

Reply:
I've found something here: https://math.stackexchange.com/questions/2156892/what-is-the-name-for-a-rectangle-that-is-not-a-square ---> Based on this it could be named "Oblong" but not sure if everyone would find it meaningful

Reply:
I've renamed it to rectangle